lib/AST/ASTContext.cpp

```diff
@@ -1683,7 +1683,7 @@ namespace {
       return OrderDeclarations(SrcMgr)(lhs, rhs);
     }
   };
-}
+} // namespace
```

Comment:
@jrose-apple What about changing to `end anonymous namespace`? That seems to be the most common form in the repo.

```
$ git grep "//.*namespace" | cut -f2 -d'}' | egrep '^ // ' | tr A-Z a-z | sort | uniq -c | sort -rn | head -10
    301  // end namespace swift
    250  // end anonymous namespace
     54  // end swift namespace
     42  // end namespace irgen
     41  // namespace swift
     26  // end anonymous namespace.
     23  // end llvm namespace
     21  // end namespace llvm
     19  // unnamed namespace
     16  // end namespace lowering
```